test/base/BaseReactor.t.sol

```diff
@@ -87,9 +87,11 @@ abstract contract BaseReactorTest is GasSnapshot, ReactorEvents, Test, DeployPer
         return (signedOrders, orderHashes);
     }
 
-    /// @dev Basic execute test, checks balance before and after
-    function testBaseExecute(uint128 inputAmount, uint128 outputAmount, uint256 deadline) public {
-        vm.assume(deadline > block.timestamp);
+    /// @dev Test of a simple execute
```

Comment:
the diff is jank because i moved all the fuzz tests to the bottom for organization.. but basically just copied the fuzz tests and made simple concrete versions for the snaps